### DISCUSSION TITLE
Palette generator as a package

### DISCUSSION THREAD

**Discussion by bartbergmans** [SimeonGriggs/tailwind-css-palette-generator#14] [Ideas]
Hi! I love the way your generator works, after trying a lot of sites, this is the only one that generates a pallete from 50 to 900 based on luminance, awesome!

I would like to use it in my application as a package, where you could give a base color and some options (for me the luminance option) and it would return an object with the generated colors. 

Are you working on something already or is it ok if I make one and link this repo?

**Comment by SimeonGriggs** [comment]
Hey thanks!

I had planned to eventually extend the API part to accept a list of parameters so you could create the same palettes from the API as you do the website ... just never got around to it yet!

**Comment by bartbergmans** [comment]
The reason I want to use it as a component is so I can use it without any latency and no dependencies on external services. Is it OK if I create a separate npm package to generate a pallette based on this repo?

**Comment by SimeonGriggs** [comment]
Sure, just include an attribution and back link somewhere 😊

**Comment by bartbergmans** [comment]
I created the package. Let me know if the attributions and back links are ok!
And feel free to edit anything 😄 

[https://github.com/bartbergmans/Palettey](https://github.com/bartbergmans/Palettey)

**Reply by SimeonGriggs** [reply]
All good. Thanks mate!
